tensorflow/contrib/cmake/CMakeLists.txt

```diff
@@ -29,9 +29,22 @@ option(tensorflow_BUILD_ALL_KERNELS "Build all OpKernels" ON)
 option(tensorflow_BUILD_CONTRIB_KERNELS "Build OpKernels from tensorflow/contrib/..." ON)
 option(tensorflow_BUILD_CC_TESTS "Build cc unit tests " OFF)
 option(tensorflow_BUILD_PYTHON_TESTS "Build python unit tests " OFF)
-option(tensorflow_BUILD_SHARED_LIB "Build TensorFlow as a shared library" OFF)
-option(tensorflow_OPTIMIZE_FOR_NATIVE_ARCH "Enable compiler optimizations for the native processor architecture (if available)" ON)
+option(tensorflow_BUILD_SHARED_LIB "Build TensorFlow as a shared library" ON)
```

Comment:
Why change the default here?

I would think that we should leave all the defaults to reflect the previous behavior.